specification/databricks/resource-manager/Microsoft.Databricks/preview/2023-09-15-preview/databricks.json

```diff
@@ -0,0 +1,1921 @@
+        "modelAsString": true
+      }
+    },
+    "ManagedIdentityConfiguration": {
+      "type": "object",
+      "description": "The Managed Identity details for storage account.",
+      "properties": {
+        "principalId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The objectId of the Managed Identity that is linked to the Managed Storage account.",
+          "readOnly": true
+        },
+        "tenantId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The tenant Id where the Managed Identity is created.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of Identity created. It can be either SystemAssigned or UserAssigned.",
+          "readOnly": true
+        }
+      }
+    },
+    "EnhancedSecurityComplianceDefinition": {
+      "type": "object",
+      "description": "Status of settings related to the Enhanced Security and Compliance Add-On.",
+      "properties": {
+        "automaticClusterUpdate": {
+          "type": "string",
+          "description": "Status of automated cluster updates feature.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "AutomaticClusterUpdate"
+          }
+        },
+        "complianceSecurityProfile": {
+          "type": "string",
+          "description": "Status of Compliance Security Profile feature.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "ComplianceSecurityProfile"
+          }
+        },
+        "complianceStandards": {
+          "description": "Compliance standards associated with the workspace.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ComplianceStandardDefinition"
+          }
+        },
+        "enhancedSecurityMonitoring": {
+          "type": "string",
+          "description": "Status of Enhanced Security Monitoring feature.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "EnhancedSecurityMonitoring"
+          }
+        }
+      }
+    },
+    "ComplianceStandardDefinition": {
+      "description": "Compliance standard that can be associated with a workspace.",
+      "type": "string",
+      "enum": [
+        "HIPAA",
+        "PCI",
+        "FEDRAMP_MODERATE"
+      ],
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "ComplianceStandard"
+      }
+    },
+    "DefaultCatalogDefinition": {
+      "type": "object",
+      "description": "Information regarding the catalog associated with the workspace.",
+      "properties": {
+        "initialType": {
```

Comment:
You need add initialName as well to the specification.

Reply:
According to the latest version of the Google Doc with Databricks, initialName should be included in the GA API version but not the public preview version.

This is an excerpt:

_Both parameters will be present in the new API version, but backend service will ONLY support the “initialType” parameter in the initial phase (ETA the week of Sept 11) and the support for “initialName” will be added as a fast follow up (ETA end of Sept or early Oct). Requests with the “initialName” parameter set during the initial phase will be rejected._

Reply:
Added the parameter initialName based on our discussion